src/redis/driver.go

```diff
@@ -17,10 +19,31 @@ type Client interface {
 	// @param args supplies the additional arguments.
 	DoCmd(rcv interface{}, cmd, key string, args ...interface{}) error
 
+	// PipeAppend append a command onto the pipeline queue.
+	//
+	// @param pipeline supplies the queue for pending commands.
+	// @param rcv supplies receiver for the result.
+	// @param cmd supplies the command to append.
+	// @param key supplies the key to append.
+	// @param args supplies the additional arguments.
+	PipeAppend(pipeline Pipeline, rcv interface{}, cmd, key string, args ...interface{}) Pipeline
```

Comment:
`PipeAppend ` seems no need to be method of `Client` interface. Add it just for unit testing. Need more thinking.